### DISCUSSION TITLE
Do we have a command to merge the pull request ?

### DISCUSSION THREAD

**Discussion by anshadmohammed** [microsoft/PowerShellForGitHub#339] [General]
Do we have a command to merge the pull request ?

**Comment by HowardWolosky** [comment]
It's [under development](https://github.com/microsoft/PowerShellForGitHub/issues/333#issuecomment-881063898).  You can see what's currently implemented [here](https://github.com/HowardWolosky/PowerShellForGitHub/blob/pullRequests/GitHubPullRequests.ps1#L1113).
